Panic when -entrypoint passed without args

```
DAYTONA - 2019/08/27 19:46:11 Will exec:  []
panic: runtime error: index out of range

goroutine 1 [running]:
main.main()
	/go/src/github.robot.car/cruise/gulp/vendor/github.com/cruise-automation/daytona/cmd/daytona/main.go:181 +0xc03
```
Here's the panicking code in `main.go`.
```go
if config.Entrypoint {
    args := flag.Args()
    log.Println("Will exec: ", args)
    binary, err := exec.LookPath(args[0]) // <<<< panic is here. args list is empty
    if err != nil {
        log.Fatalf("Error finding '%s' to exec: %s\n", args[0], err)
    }
    err = syscall.Exec(binary, args, os.Environ())
    if err != nil {
        log.Fatalf("Error from exec: %s\n", err)
    }
}
```

--- a/cmd/daytona/main.go
+++ b/cmd/daytona/main.go
@@ -189,6 +189,9 @@ func main() {
 
 	if config.Entrypoint {
 		args := flag.Args()
+		if len(args) == 0 {
+			log.Fatalln("No arguments detected with use of -entrypoint")
+		}
 		log.Println("Will exec: ", args)
 		binary, err := exec.LookPath(args[0])
 		if err != nil {

